src/vivarium_nih_us_cvd/data/loader.py

```diff
@@ -370,74 +381,276 @@ def _get_ihd_sequela() -> Dict[str, List["Sequela"]]:
             for s in causes.ischemic_heart_disease.sequelae
             if s.name == "asymptomatic_ischemic_heart_disease_following_myocardial_infarction"
         ],
+        "heart_failure": [
+            s for s in causes.ischemic_heart_disease.sequelae if "heart_failure" in s.name
+        ],
     }
     return seq_by_cause
 
 
-def load_prevalence_ihd(key: str, location: str) -> pd.DataFrame:
+def get_heart_failure_proportions(location: str, heart_failure_type: str) -> pd.Series:
+    hf_proportions = pd.read_csv(paths.FILEPATHS.HEART_FAILURE_PROPORTIONS)
+    hf_proportions = hf_proportions.query(
+        "location_name==@location & sim_cause==@heart_failure_type"
+    )
+
+    hf_proportions = hf_proportions[PROPORTION_DATA_INDEX_COLUMNS + ["proportion"]]
+
+    return hf_proportions
+
+
+def get_proportion_adjusted_heart_failure_data(
+    location: str, heart_failure_type: str, measure: str
+) -> pd.DataFrame:
+    # pull measure data
+    location_id = utility_data.get_location_id(location)
+    heart_failure_data = gbd.get_modelable_entity_draws(
+        data_values.HEART_FAILURE_ME_ID, location_id
+    )
+    measure_data = heart_failure_data[
+        heart_failure_data.measure_id == vi_globals.MEASURES[measure]
+    ]
+    measure_data = vi_utils.normalize(measure_data, fill_value=0)
+    measure_data = measure_data.filter(
```

Comment:
Don't think you need this since you do it again anyway a few lines below after multiplying

Reply:
The second time is to remove the proportion column that got added after merging the proportion data.